.github/workflows/publish.yml

```diff
@@ -63,8 +63,14 @@ jobs:
           GH_TOKEN: ${{ github.token }}
         working-directory: src/D2L.Bmx/build
         run: |
-          Compress-Archive -Path "$env:FILE_NAME" -DestinationPath "bmx-$env:PLATFORM.zip"
-          gh release upload "$env:RELEASE_TAG" "bmx-$env:PLATFORM.zip"
+          if ($env:PLATFORM -eq 'win') {
+            Compress-Archive -Path "$env:FILE_NAME" -DestinationPath "bmx-$env:PLATFORM.zip"
+            gh release upload "$env:RELEASE_TAG" "bmx-$env:PLATFORM.zip"
+          } else if ($env:PLATFORM -eq 'osx') {
+            chmod u+x "bmx"
```

Comment:
not sure if `u+x` is enough, or if `+x` is better. I vaguely remember there was a case of `u+x` not doing enough for us. I think it was a lambda issue @boarnoah had.

Reply:
Don't recall, from what I understand re: tar, it preserves the bits for user, group and others. What it can't do is preserve ownership. So if we create a file with rw for user, but at the other end the current user is different there could be issues? 

Not sure I understood that properly. It seems like a safe bet to to ``+x`` in this case.

Reply:
> it preserves the bits for user, group and others. What it can't do is preserve ownership

That's my understanding too.

> So if we create a file with rw for user, but at the other end the current user is different there could be issues?

Not sure how a user switch would behave. I suspect tar would set the user execution flag on the other side appropriately. My concern is that there'd be a need to to have +x on the group and root too. It seems customary for "installed" executables to get +x on all three.

So yeah, agreed that `+x` is what we want.